packages/rn-tester/js/examples/KeyboardEventsExample/KeyboardEventsExample.js

```diff
@@ -13,82 +13,108 @@
 const React = require('react');
 const ReactNative = require('react-native');
 import {Platform} from 'react-native';
-const {Button, PlatformColor, StyleSheet, Text, View} = ReactNative;
+const {Button, PlatformColor, StyleSheet, Text, TextInput, View} = ReactNative;
 
 import type {KeyEvent} from 'react-native/Libraries/Types/CoreEventTypes';
 
-type State = {
-  eventStream: string,
-  characters: string,
-};
-
-class KeyEventExample extends React.Component<{}, State> {
-  state: State = {
-    eventStream: '',
-    characters: '',
-  };
-
-  onKeyDownEvent: (e: KeyEvent) => void = (e: KeyEvent) => {
-    console.log('received view key down event\n', e.nativeEvent.key);
-    this.setState({characters: e.nativeEvent.key});
-    this.setState(prevState => ({
-      eventStream:
-        prevState.eventStream + '\nKey Down: ' + prevState.characters,
-    }));
+function KeyEventExample(): React.Node {
+  const [log, setLog] = React.useState([]);
+  const appendLog = (line: string) => {
+    const limit = 12;
+    let newLog = log.slice(0, limit - 1);
+    newLog.unshift(line);
+    setLog(newLog);
   };
 
-  onKeyUpEvent: (e: KeyEvent) => void = (e: KeyEvent) => {
-    console.log('received key up event\n', e.nativeEvent.key);
-    this.setState({characters: e.nativeEvent.key});
-    this.setState(prevState => ({
-      eventStream: prevState.eventStream + '\nKey Up: ' + prevState.characters,
-    }));
-  };
-
-  render() {
-    return (
+  return (
+    <View style={{padding: 10}}>
+      <Text>
+        Key events are called when a component detects a key press.To tab
+        between views on macOS: Enable System Preferences / Keyboard / Shortcuts
+        > Use keyboard navigation to move focus between controls.
+      </Text>
       <View>
-        <Text>Key events are called when a component detects a key press.</Text>
-        <View>
-          {Platform.OS === 'macos' ? (
+        {Platform.OS === 'macos' ? (
+          <>
+            <Text style={styles.title}>View</Text>
+            <Text style={styles.text}>
+              validKeysDown: [g, Escape, Enter, ArrowLeft]{'\n'}
+              validKeysUp: [c, d]
+            </Text>
```

Comment:
Could you actually move these examples to the TextInput page? I think it makes more sense to be there